sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/utils.py

```diff
@@ -46,7 +46,7 @@ def dst(self, dt):
 
 
 try:
-    from datetime import timezone  # pylint: disable=ungrouped-imports
+    from datetime import timezone
```

Comment:
same here from the other PR ? perhaps vendor update??

Reply:
this should be the same now as the other pr